config.yaml

```diff
@@ -0,0 +1,17 @@
+asrr:
+  s3:
+    region:
+    endpoint:
+    bucket:
+    access_key:
+    secret_key:
+s3:
```

Comment:
What's the difference between the `s3` and `asrr.s3` stanzas?

Reply:
@DoctorVin s3 is the destination bucket. asrr s3 is the source bucket for AsRockrack firmware images. It's a hack because our firmware images for ASRR come from a ftp site that's not available publicly so at the time this was added we agreed to keep the fw images in that s3 bucket and sync from there instead of using the private ftp they provide.